### DISCUSSION TITLE
Is this project being maintained?

### DISCUSSION THREAD

**Discussion by acgetchell** [onqtam/doctest#814] [General]
I opened an issue [here](https://github.com/doctest/doctest/issues/813), but perhaps a discussion is warranted instead.

As noted, there hasn't been a commit on any branch since March of 2023, and the roadmap dates from 2022.

This is a really useful project -- is there something we can do to support it?

**Comment by onqtam** [comment]
Looking for maintainers: https://github.com/doctest/doctest/issues/554#issuecomment-1740568920

**Comment by onqtam** [comment]
closing this discussion in favor of the linked issue (that is also now pinned in the issues section)
https://github.com/doctest/doctest/issues/554#issuecomment-1740568920

**Comment by onqtam** [comment]
Actually, I'll reopen this in order to draw more attention to the issue of finding a maintainer:
https://github.com/doctest/doctest/issues/554#issuecomment-1740568920
